content/governance.md

```diff
@@ -70,11 +70,13 @@ The current team members are:
 * Tobias Schmidt ([SoundCloud](https://soundcloud.com/))
 * Tom Wilkie ([Grafana Labs](https://grafana.com/))
 
-The ex-members who are publicly listed are:
+Previous members:
```

Comment:
Note that I removed the shenanigans about "publicly listed" as per the discussion on #1581 .